### PR TITLE
deposit all bveCVX lp into bbveCVX

### PR DESCRIPTION
**Pull request by jalbrekt85**
**#156**

### CODE REVIEW

**Review by jorijnsmit — COMMENTED** [review]
can you add snapshots so we can verify it visually at the end?

**Review by jorijnsmit — APPROVED** [review]
script does what it needs to do, except for fails in the call trace:
```
    │           └── SettV4h.depositAll  [DELEGATECALL]  2104:2504  [84990 / 8274 gas]
    │               │   ├── address: 0x0B7Cb84bc7ad4aF3E1C5312987B6E9A4612068AD
    │               │   ├── input arguments: None
    │               │   └── revert reason: Access denied for caller
```

 we will have to whitelist treasury vault msig first on bbvecvxcvx before running this. i still suggest to merge it though, and have it ready to post.

File: scripts/issue/156/issue156.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+
+def main():
+    """
+    deposit all bveCVX lp to bbveCVX sett
+    """
+
+    safe = GreatApeSafe(registry.eth.badger_wallets.treasury_vault_multisig)
+
+    bveCVX = interface.ICurveLP(
```
Comment (petrovska-petro) [inline scripts/issue/156/issue156.py:13]: The variable naming was confusing for me, mixing it up with the real `bvecvx` sett token, could this be renamed to reflect the bvecvx_cvx_lp instead?

Comment (jalbrekt85) [reply scripts/issue/156/issue156.py:13]: agreed, these token names can get confusing

**Review by jorijnsmit — APPROVED** [review]

**Review by petrovska-petro — APPROVED** [review]

**Pull request merged**